app/assets/stylesheets/code-editor.scss

```diff
@@ -54,9 +54,19 @@ body .md-editor {
         text-decoration: underline;
     }
 
-    pre {
+    pre:not([class*="language-"]) {
```

Comment:
> In the light theme, code blocks in discussions had their colors overriden to look like a normal `<pre>` tag. I assume this wasn't an intended feature and thus I've done 5630c26. Please let me know if it was otherwise.

That was intentional, so please reverse this one.

Reply:
Thanks.